pipelines/components/tensorflow/kmeansclustering/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM tensorflow/tensorflow
+
+USER root
+
+WORKDIR /
+
+# To allow running it locally.  See local_run.sh.
+RUN touch /credentials.json
+
+ADD *.py /
+
+ENTRYPOINT ["python", "work.py"]
```

Comment:
Should we all agree on the same python filename for the entrypoint? For instance mine is task.py while yours is work.py.